whitelabelco/sofo-foods
# Cp/feat/about copy

### Description 💻

Resolves: #214 & #215 

### Implementation 💡

Fixes width styling on Hero component to allow larger paragraphs to expand with screen
updates copy on About Us page


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click About Us
- confirm content updates match the tickets

### Screenshots/Demos 📷

### Checklist ✅
